mp/src/game/server/logic_random_outputs.h

```diff
@@ -0,0 +1,51 @@
+//========= Copyright � 1996-2005, Valve Corporation, All rights reserved. ============//
+//
+// Purpose: 
+//
+// $NoKeywords: $
+//
+//=============================================================================//
+#ifndef LOGICRANDOMOUTPUTS_H
+#define LOGICRANDOMOUTPUTS_H
```

Comment:
You can replace the ifdef guards with just `#pragma once` 

You can also get rid of the empty comment blocks.